xml/System.Diagnostics/ActivitySource.xml

```diff
@@ -110,7 +110,7 @@
       <Parameters />
       <Docs>
         <summary>Checks if there are any listeners for this activity source.</summary>
-        <returns>To be added.</returns>
+        <returns>true if there is a listener registered for this activity source. false otherwise.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if there is a listener registered for this activity source; otherwise, <see langword="false" />.</returns>
```